index.bs

```diff
@@ -165,6 +169,92 @@ To <dfn abstract-op lt="set-dest">set the `Sec-Fetch-Dest` header</dfn> for a [=
 </div>
 
 
+The `Sec-Fetch-Frame-Ancestors` HTTP Request Header {#sec-fetch-frame-ancestors-header}
+---------------------------------------------------------------------------------------
+
+The <dfn http-header export>`Sec-Fetch-Frame-Ancestors`</dfn> HTTP request header exposes the
+relationship between a [=request=]'s initiator's origin, its target origin, and the origins of
```

Comment:
> its target origin

I know what this means, but that phrasing isn't used anywhere in the Fetch spec, so I think it's less clear than it could be. Let's say something like `its [=request/url=]'s [=url/origin=]` instead.

Same comment for the other header's introduction.